docs/management/cases/setup-cases.asciidoc

```diff
@@ -40,3 +40,8 @@ least once, which creates a user profile.
 |=== 
 
 For more details, refer to <<kibana-privileges>>.
+
+NOTE: If you are using an on-premises {kib} deployment and you want the email
+notifications and the external incident management systems to contain
```

Comment:
Should we add how to configure the email notifications? Ideally, we should link to a section regarding notifications but I cannot find it. @gsoldevila Do you know if there is a page in the Kibana docs about notification configuration?

User should do the following on-premises to configure notifications:

```
notifications.connectors.default.email: 'notifications-email'

xpack.actions.preconfigured:
  notifications-email:
    name: 'My name'
    actionTypeId: .email
    config:
      service: other
      from: test@example.com
      host: example.com
      port: 1025
      secure: false
      hasAuth: false
```

Reply:
Yup, I was going to do that in a separate PR after testing it in a Cloud environment, if that's okay.